### ISSUE TITLE
Improve accessibility for users with screen readers

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#8371]
## Description 
User reports:
> I'm using the Windows browser ... several unlabeled buttons appear in the UI. 

I believe these are likely the native UI elements (back/foward/reload/bookmark). @petemill has shown me that view elements are inspectable on Windows, so finding these should be fairly easy. Once found, we can provide a proper label
